presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveFileFormats.java

```diff
@@ -662,6 +684,9 @@ else if (testColumn.getObjectInspector().getTypeName().equals("timestamp")) {
                         SqlTimestamp expectedTimestamp = new SqlTimestamp((Long) expectedValue, SESSION.getTimeZoneKey());
                         assertEquals(actualValue, expectedTimestamp, "Wrong value for column " + testColumn.getName());
                     }
+                    else if (testColumn.getObjectInspector().getTypeName().startsWith("char")) {
+                        assertEquals(((String) actualValue).trim(), expectedValue, "Wrong value for column " + testColumn.getName());
```

Comment:
right. expected value (which is internal char `Slice`) is now padded with spaces.